app/actions/balancesActions.js

```diff
@@ -21,23 +21,42 @@ async function getBalances({ net, address, tokens }: Props) {
     endpoint = await api.getRPCEndpointFrom({ net }, api.neoscan)
   }
 
-  // token balances
-  const tokenBalances = await api.nep5.getTokenBalances(
-    endpoint,
-    tokens
-      .filter(token => !token.isUserGenerated)
-      .map(token => token.scriptHash),
-    address
+  const chunks = tokens
+    .filter(token => !token.isUserGenerated)
+    .reduce((accum, currVal, index) => {
+      if (!accum.length) {
+        accum.push([currVal.scriptHash])
+        return accum
+      }
+
+      if (accum[accum.length - 1].length < 4) {
+        accum[accum.length - 1].push(currVal.scriptHash)
```

Comment:
this push does not have an array surrounding currVal.scripthash but the rest do

Reply:
Thats because its pushing into an existing arr... Think of the structure like this:
`[ [hash1, hash2, hash3, hash4], [hash5, hash6, hash7, hash8] ... ]`